src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const allPop = document.querySelectorAll('.population');
+const arrPop = [...allPop].map(number =>
```

Comment:
`allPop` and `arrPop` look almost the same so can be easily confused in future.
Try to avoid such situations and provide concise, informative names for your variables.

Reply:
Ok, got it